spec/unit/server_spec.rb

```diff
@@ -194,40 +194,25 @@ def app
 
     context "broken description" do
       before do
-        store_raw_description("foo", <<-EOT
-        {
-            "environment": {
-                "locale": "C"
-            },
-            "os": {
-                "name": "openSUSE 13.1 (Bottle)",
-                "version": "13.1 (Bottle)",
-                "architecture": "x86_64"
-            },
-            "unmanaged_files": {
-                "_attributes": {
-                    "extracted": true,
-                    "has_metadata": false,
-                    "foo": true
-                },
-                "_elements": []
-            },
-            "meta": {
-                "environment": {
-                    "modified": "2016-04-05T12:07:33+02:00",
-                    "hostname": "example.com"
-                },
-                "os": {
-                    "modified": "2016-04-05T12:07:33+02:00",
-                    "hostname": "example.com"
-                },
-                "unmanaged_files": {
-                    "modified": "2016-04-05T12:07:33+02:00",
-                    "hostname": "example.com"
-                }
+        store_raw_description(
+          "foo", <<-EOT
+            {
+              "unmanaged_files": {
+                  "_attributes": {
+                      "extracted": true,
```

Comment:
You were right about the validation, this was my mistake. In this case it's fine to use ```store_raw_description``` here. 
Just strip down the example JSON to the utmost necessary attributes only, which means you can also get rid of the ```extracted``` attribute, the ```_elements``` array and everything inside the ```meta``` scope. Other than that it ltgm!